src/main.rs

```diff
@@ -41,6 +53,25 @@ enum Commands {
         /// If this path is provided and it exists, the private key is read from this file and used, if it does not exist the private key will be persisted to this location.
         #[clap(long)]
         key: Option<PathBuf>,
+        /// Optional rpc port, defaults to 4919
+        #[clap(long)]
+        rpc_port: Option<u16>,
+    },
+    /// List hashes
+    #[clap(about = "List hashes")]
```

Comment:
I belive the `about` is not needed, it will be parsed by default from the doc comment

Reply:
All the things have `about =`.